pages/lunchweek/[slug].tsx

```diff
@@ -0,0 +1,73 @@
+import { ReactMarkdown } from "react-markdown/lib/react-markdown";
+
+import { Button } from "../../components/common";
+import PageLayout from "../../components/common/layout/PageLayout";
+import Background from "../../components/sections/lunchweek/Background";
+import SectionWrapper from "../../components/common/layout/SectionWrapper";
+import IndividualHero from "../../components/sections/lunchweek/IndividualHero";
+
+import { SanityChangelog, SanityFooter, SanityNavigation, SanitySeo } from "../../types/schema";
+import { getCommonData, getChangelogBySlug, getLatestChangelogsExceptSlug } from "../../lib/sanity";
+import Launch from "../../components/sections/lunchweek/Launch";
+
+export async function getServerSideProps({ params } : { params: { slug: string }}) {
+  if (!params.slug) { 
+    return { notFound: true };
+  }
+
+  const changelog = await getChangelogBySlug(params.slug);
+
+  // there's no associated log to the slug given
+  if (!changelog) {
+    return { notFound: true };
+  }
+
+  const commonData = await getCommonData();
+  const latestChanges = await getLatestChangelogsExceptSlug(params.slug);
+
+  return { props: { changelog, commonData, latestChanges }};
+}
+
+export default function ChangelogPage({ changelog, commonData, latestChanges } : { 
+  changelog: SanityChangelog,  
+  commonData: {
+    navigationLinks: SanityNavigation[],
+    seoData: SanitySeo,
+    footer: SanityFooter[]
+  },
+  latestChanges: SanityChangelog[],
+}) {
+
+  return (
+    <PageLayout
+      seoData={commonData.seoData}
+      navigationURLs={commonData.navigationLinks}
+      BackgroundWrapper={Background}
+    >
+      <IndividualHero 
+        title={changelog.title ?? "Test"} 
+        topics={changelog.topics ?? []} 
+        date={changelog.date ?? "2001-02-02"} 
+        url={`https://opensauced.pizza/changelog/${changelog.slug?.current}`}
+      /> 
+      <ReactMarkdown
+        className="prose-invert mx-auto mb-24 leading-loose prose prose-md prose-img:mx-auto prose-img:rounded-md"
+      >
+        {changelog.changelogContent}
+      </ReactMarkdown>
+
+      <SectionWrapper>
+        <section className="flex flex-col py-24 border-t-2 border-brandOrange">
+          {latestChanges.length > 0 && latestChanges.map((changelog, index) => (
+            <Launch count={latestChanges.length} index={index} key={changelog._id} changelog={changelog}/>
```

Comment:
`changelog` needs to be added in the `LaunchItemsProps`. This is causing the Netlify deploy to fail. https://app.netlify.com/sites/opensauced-landing/deploys/664ad65a8ba964000814dbc2